libs/initScripts/local.js

```diff
@@ -149,6 +196,149 @@ const run = async () => {
 
 run()
 
+/*
+  Helper function to bootstrap additional local content nodes onto the L2 local network
+  Assumes 3 content nodes are already up prior to execution
+*/
+const addL2ContentNode = async (
+  audiusLibs,
+  ethAccounts,
+  newCnodeId,
+  newCnodeDelegateWallet
```

Comment:
I noticed that we renamed this variable at some other parts, might be nice to rename it everywhere moving forward.